modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -78,6 +78,29 @@ public function __construct($info, $config) {
 		}else{
 			throw new Exception("cas login URL not specified");
 		}
+
+
+		// Check for the required ldap configuration 
+		if(!isset($this->_ldapConfig['servers'])) {
```

Comment:
After a bit of testing it looks like the _ldapconfig['servers'] is not a required field for CAS authentication to work, but the ldap array is required in the config. That is why I was thinking the servers attribute was required.

```php
'example-cas' => array(
    'cas:CAS',
    'cas' => array(
        'login' => 'https://cas.example.edu/login',
        'validate' => 'https://cas.example.edu/validate',
        'logout' => 'https://cas.example.edu/logout'
    ),
    'ldap' => array(),
),
```